sdk/management/pom.xml

```diff
@@ -199,6 +199,79 @@
                 </executions>
             </plugin>
 
+            <plugin>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
+                <version>3.1.12.2</version> <!-- {x-version-update;com.github.spotbugs:spotbugs-maven-plugin;external_dependency} -->
+                <dependencies>
+                <dependency>
+                    <groupId>com.azure</groupId>
+                    <artifactId>sdk-build-tools</artifactId>
+                    <version>1.0.0</version> <!-- {x-version-update;com.azure:sdk-build-tools;external_dependency} -->
+                </dependency>
+                <dependency>
+                    <groupId>com.github.spotbugs</groupId>
+                    <artifactId>spotbugs</artifactId>
+                    <version>4.0.0-beta3</version> <!-- {x-version-update;com.github.spotbugs:spotbugs;external_dependency} -->
```

Comment:
is the dependency version managed by parent pom?

Reply:
Yes. For data-plane sdk, all plugins are managed by parent pom. https://github.com/azure/azure-sdk-for-java/blob/340ce8d5da791d4f21fa5ec35ece8832b490f8bf/sdk/parents/azure-client-sdk-parent/pom.xml#L176